ethapi/committee_cert_json.go

```diff
@@ -0,0 +1,40 @@
+package ethapi
+
+import (
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/bls"
+	"github.com/0xsoniclabs/sonic/scc/cert"
+)
+
+type committeeCertificateJson struct {
+	ChainId   uint64                    `json:"chainId"`
+	Period    uint64                    `json:"period"`
+	Members   []scc.Member              `json:"members"`
+	Signers   cert.BitSet[scc.MemberId] `json:"signers"`
+	Signature bls.Signature             `json:"signature"`
+}
+
+func (c committeeCertificateJson) ToCommitteeCertificate() cert.Certificate[cert.CommitteeStatement] {
+	aggregatedSignature := cert.NewAggregatedSignature[cert.CommitteeStatement](c.Signers, c.Signature)
+
+	newCert := cert.NewCertificateWithSignature(
+		cert.NewCommitteeStatement(
+			c.ChainId,
+			scc.Period(c.Period),
+			scc.NewCommittee(c.Members...)),
+		aggregatedSignature)
+
+	return newCert
+}
+
+func CommitteeCertificateToJson(c cert.Certificate[cert.CommitteeStatement]) committeeCertificateJson {
```

Comment:
nit: after #65 the parameter could be a `cert.CommitteeCertificate`

Reply:
if you don't mind, we can merge #65 first, and then use those types. I would prefer it that way